static.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Is this nginx configuration?

Reply:
The static server Heroku buildpack turns this into nginx configuration. So, indirectly, yes.